Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Possible_External_RDP_Brute-Force.yml

```diff
@@ -0,0 +1,3115 @@
+id: Cortex XDR - Possible External RDP Brute-Force
```

Comment:
Go over all tasks/playbooks that aren't CommonPlaybooks/Scripts and mark the check box of 'skipunavailable'
<img width="605" alt="image" src="https://user-images.githubusercontent.com/28757135/213924141-b5ac89c1-0e95-4409-ba73-32d31ffa009b.png">